src/scss/components/_modal-waitlist.scss

```diff
@@ -0,0 +1,34 @@
+.waitlist-modal {
+  .close {
+    font-size: $font-size-xs;
+    color: $sh-purple;
+    opacity: 1;
+
+    &:hover {
+      color: $sh-purple;
+    }
+  }
+
+  .masking-select {
+    color: $gray-500;
+    padding: $custom-select-padding-y ($custom-select-padding-x + $custom-select-indicator-padding) $custom-select-padding-y $custom-select-padding-x/1.5;
+    border: $custom-select-border-width solid $custom-select-border-color;
+    background: $custom-select-bg $custom-select-background-mask;
+    width: calc(100% - #{$grid-gutter-width});
+
+    &.selected {
+      color: $custom-select-color;
+    }
+
+    @include border-radius($custom-select-border-radius, 0);
+    @include box-shadow($custom-select-box-shadow);
+
+    @include media-breakpoint-up('lg') {
+      width: calc(100% - #{$grid-gutter-width-lg});
+    }
+  }
```

Comment:
we need to be consistent and use the styles that we have already for all Form components, including the SELECT

Reply:
actually this is only for masking custom select element, and i already used what we have in the form except for width because position is absolute so i need to reduce width by gutter